### PR TITLE
fix: return unconfirmed balances as part of account balance response

### PR DESCRIPTION
**Pull request by kaladinlight**

### CODE REVIEW

**Review by 0xdef1cafe — APPROVED** [review]

File: packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts [hunk 1]
```diff
@@ -115,8 +116,10 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       const { chain, network } = caip2.fromCAIP2(caip)
       const { data } = await this.providers.http.getAccount({ pubkey: pubkey })
 
+      const balance = bnOrZero(data.balance).plus(bnOrZero(data.unconfirmedBalance))
+
       return {
-        balance: data.balance,
+        balance: balance.toString(),
```
Comment (asamere) [inline packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: is this always safe i.e. return as decimal? concern toString() will result in exponential notation with very large/precise numbers.

Comment (asamere) [reply packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: is balance an int (base units) here?

Comment (asamere) [reply packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: `balance.toFixed(n)`?

Comment (kaladinlight) [reply packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: Hmm... possible it defaults to exponential notation at large numbers. `toString()` being used heavily throughout whole stack to my knowledge, so we would probably need to address everywhere in another PR?

Comment (kaladinlight) [reply packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: looks like `toFixed()` is the trick. Think we need a chore PR to update throughout stack for sure.

Comment (asamere) [reply packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:122]: Ok, thanks!

**Review by asamere — APPROVED** [review]

**Pull request merged**